cc/cxx_standard/cxx_standard.bzl

```diff
@@ -0,0 +1,78 @@
+# Copyright 2025 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Globals for C++ std version flags."""
+
+load("//cc/compiler:compilers.bzl", "COMPILERS")
+
+VERSIONS = [
+    "98",
+    "03",
+    "11",
+    "14",
+    "17",
+    "20",
+    "23",
+    "26",
+    "2c",
```

Comment:
What's the backwards compatibility story here? Could we ever delete this constant once we have added it?